sdk/tables/data-tables/src/generated/operations/table.ts

```diff
@@ -256,14 +205,10 @@ export class Table {
     table: string,
     options?: TableGetAccessPolicyOptionalParams
   ): Promise<TableGetAccessPolicyResponse> {
-    const operationArguments: coreHttp.OperationArguments = {
-      table,
-      options: coreHttp.operationOptionsToRequestOptionsBase(options || {})
-    };
-    return this.client.sendOperationRequest(
-      operationArguments,
+    return (this.client.sendOperationRequest(
+      { table, options },
       getAccessPolicyOperationSpec
-    ) as Promise<TableGetAccessPolicyResponse>;
+    ) as unknown) as Promise<TableGetAccessPolicyResponse>;
```

Comment:
I wonder why casting to `unknown` first? the source and target types are not compatible?

Reply:
oh this isn't needed anymore! I'll remove it. Thanks for pointing it out! 👍